src/components/Map/Map.js

```diff
@@ -0,0 +1,221 @@
+import React, {Component} from "react";
+import {connect} from "react-redux";
+import {withStyles} from "@material-ui/core/styles";
+import withWidth from "@material-ui/core/withWidth/index";
+import mapboxgl from "mapbox-gl";
+import "mapbox-gl/dist/mapbox-gl.css";
+import "./Map.css";
+import {
+  findMyLocation,
+  setBottomDrawerData,
+  toggleBottomDrawer,
+  setMap,
+  selectMapItem
+} from "../../redux/actions";
+import {
+  FIND_MY_LOCATION_ERROR,
+  FIND_MY_LOCATION_OUT_OF_BOUNDS,
+  FIND_MY_LOCATION_SELECT,
+  FIND_MY_LOCATION_SUCCESS
+} from "../../redux/constants";
+
+mapboxgl.accessToken =
```

Comment:
Should we pull this out into ./config.js? Just incase we ever have multiple map components across the project and I think would be easier to find/update in the future if needed or have a dev/prod specific key if the client wanted to pay for some mapbox feature.

Reply:
Yeah good idea